universal-login-wallet/src/config/config.ts

```diff
@@ -20,6 +22,7 @@ export default Object.freeze({
   },
 
   production: {
+    network: process.env.NETWORK as Network,
```

Comment:
use asNetwork cast (restless)

Reply:
Wont work since we only want to perform the cast when we are running on production. In this code the cast will be attempted even if we're running in dev or test and the app will crash